Didi/obsolete listview (#990)
telerik/maui-docs: controls/listview/xamarin-migration.md

position: 20
---

> >caution ListView control is now obsolete and will be removed in future. Use `RadCollectionView` instead. The [CollectionView]({%slug collectionview-overview%}) is a complete rewrite of the ListView from the ground up. The CollectionView offers improved performance, enhanced features, and a modernized approach to managing lists of data. The CollectionView incorporates all key features of the ListView. You can read about the differences between both components and how to migrate to the new CollectionView in the following article: [Migrating the Telerik .NET MAUI ListView to CollectionView]({%slug listview-migration%}).

Reviewer: ```suggestion
>caution The ListView control is now obsolete and will be removed in the future. Use the [CollectionView]({%slug collectionview-overview%}) control instead. The CollectionView is a complete, ground-up rewrite of the ListView. The CollectionView offers improved performance, enhanced features, and a modernized approach to managing lists of data. The CollectionView incorporates all of the ListView's key features. More about the differences between both components and how to migrate to the new CollectionView is available in the [Migrating the Telerik .NET MAUI ListView to CollectionView]({%slug listview-migration%}) article.
```

Author: @yordan-mitev thank you for the review. this is just a draft, I will move the message in template, once we merge the revamp updates.
